README.md

```diff
@@ -42,11 +49,29 @@ let dog = new Dog("Shadow");
 dog.speak(); // Shadow says woof!
 ```
 
+( KG: Code like this needs a post-game explanation of what it is doing.)
+
+In our code, we have 2 `class`es: `Pet` and `Dog`. The `Dog` class is a
+_subclass_ or _child_ `class` of `Pet` and it uses the `extends` keyword to
+inherit methods (in this case the `constructor`) from the parent `class`.
+Because the child (`Dog`) class extends the parent (`Pet`), `Dog`'s constructor
+is `Pet`'s constructor.
+
 ## Use the `super` Method
 
-In our code, we have 2 JavaScript `class`es: `Pet` and `Dog`. The `Dog` class
-is a _subclass_ or _child_ `class` of `Pet` and it uses the `extends` keyword
-to inherit methods from the parent `class`. With `super`, we can inherit
+(KG: I think this is overloaded, too big. Maybe we can turn this Readme into 3
+smaller readmes labs (still using the Pet and Dog and Bird domain) but discuss and
+then test the following
```

Comment:
@drakeltheryuujin @maxwellbenton here's where i see opportunity to break out into smaller lessons. they could probably all be labs with just one test scenario.